source/fundamentals/transactions.txt

```diff
@@ -160,89 +240,95 @@ order details. Suppose you organize the data updates as follows:
-    { name: "beach towel", sku: 7865, qty: 41 }
-   ]
+   { item: "sunblock", item_id: 5432, qty: 85 },
+   { item: "beach towel", item_id: 7865, qty: 41 }
 
 The code examples also perform operations on the ``orders`` collection, but
-do not require any prior sample documents.
+this collection does not contain any data before the transaction is
+performed.
 
 The code examples use the ``cart`` and ``payment`` variables to represent
-a sample list of items purchased and the order payment details as follows:
+a sample list of items purchased and the order payment details. The
+following code describes the contents of the ``cart`` and ``payment`` variables:
 
 .. code-block:: javascript
 
    const cart = [
-     { name: 'sunblock', sku: 5432, qty: 1, price: 5.19 },
-     { name: 'beach towel', sku: 7865, qty: 2, price: 15.99 }
+     { item: 'sunblock', item_id: 5432, qty: 1, price: 5.19 },
+     { item: 'beach towel', item_id: 7865, qty: 2, price: 15.99 }
    ];
    const payment = { customer: 98765, total: 37.17 };
 
 .. important::
-
-   The examples in the following sections require that you create the
-   collections outside of the transaction or that you are using MongoDB 4.4 or
-   later. For more information on creating collections inside a transaction,
-   see the
-   :manual:`Create Collections and Indexes in a Transaction </core/transactions/#std-label-transactions-create-collections-indexes>`
-   server guide.
+   
+   If you are connected to a deployment running MongoDB Server version
+   4.2 or earlier, you can perform write operations in a transaction
+   only on collections that already exist. Starting in Server version 4.4, the
+   server automatically creates collections when you perform operations
+   on them in a transaction. To learn more about this behavior,
+   see :manual:`Create Collections and Indexes in a Transaction
+   </core/transactions/#create-collections-and-indexes-in-a-transaction>`
+   in the Server manual.
 
 .. _nodejs-transaction-core-api-example:
 
 Core API Implementation
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-The code example in this section demonstrates how you can use the Core API to
-run the multi-document payment transaction in a session. This function
-shows how you can perform the following:
+The code example in this section demonstrates how to use the Core API to
+perform a multi-document transaction in a session. In this example, the
+transaction reflects the actions needed when a customer purchases items from
+your store.
+
+This example shows how to perform the following actions:
 
 1. Start a session
-#. Start a transaction, specifying transaction options
-#. Perform data operations in the same session
-#. Commit a transaction, or cancel it if the driver encounters an error
-#. End a session
+#. Start a transaction with transaction options
+#. Perform read and write operations in the session to record the purchase
+#. Commit the transaction, or end it if the driver encounters an error
+#. End the session
```

Comment:
Suggestion:
I think it would be good to keep consistent with indefinite articles throughout the list. The prior items refer to an indefinite session and transaction, so the rest should be similar, e.g.:

"Perform read and write operations in a session
Commit a transaction or end it...
End a session"

Reply:
Revising this whole section to be more specific